zerver/tests/test_subs.py

```diff
@@ -1078,6 +1079,9 @@ def test_add_and_remove_stream_as_default(self) -> None:
         stream = self.make_stream("stream", realm=realm)
         stream_id = self.subscribe(user_profile, "stream").id
 
+        def fresh_stream() -> Stream:
+            return get_stream_by_id_in_realm(stream_id, realm)
```

Comment:
I think we're quibbling about names.  The "fresh_stream" name is both accurate and appropriate.  You want a totally freshly fetched Stream object from the database to properly validate that the round-trip POST or whatever worked.  Otherwise you're gonna get either a false negative or false positive.

Reply:
If we made `refresh_user`, I don't see why we wouldn't want this to be parallel.